src/test/resources/features/Curriculum.feature

```diff
@@ -0,0 +1,50 @@
+Feature: creates curriculums
```

Comment:
Please put the 'c' of creates with upperCase and with the verb in base form: 'Create'

Reply:
done